src/tests/functional/shared_test_classes/src/single_op/dft.cpp

```diff
@@ -49,19 +45,29 @@ void DFTLayerTest::generate_inputs(const std::vector<ov::Shape>& targetInputStat
 
     inputs.clear();
     Tensor data_tensor = ov::test::utils::create_and_fill_tensor_act_dft(funcInput->get_element_type(),
-                                            targetInputStaticShapes[0],
-                                            inGenData.range, inGenData.start_from, inGenData.resolution, inGenData.seed);
+                                                                         targetInputStaticShapes[0],
+                                                                         inGenData.range,
+                                                                         inGenData.start_from,
+                                                                         inGenData.resolution,
+                                                                         inGenData.seed);
     inputs.insert({funcInput->get_node_shared_ptr(), data_tensor});
+
+    const bool parameters_only = axes_in_type == utils::InputLayerType::PARAMETER;
+    if (axes_in_type == utils::InputLayerType::PARAMETER) {
+        auto tensor_axes = ov::Tensor(ov::element::i64, ov::Shape({axes.size()}));
+        std::memcpy(tensor_axes.data(), &axes[0], axes.size() * sizeof(int64_t));
+        inputs.insert({funcInputs[1].get_node_shared_ptr(), tensor_axes});
+    }
+
+    if (size_in_type == utils::InputLayerType::PARAMETER && signal_size.size() > 0) {
```

Comment:
```suggestion
    if (parameters_only && signal_size.size() > 0) {
```

Reply:
`parameters_only` is a leftover variable (to be removed), the updated logic has separate flags for axis and sizes, to test more combinations